static/bidder-info/admatic.yaml

```diff
@@ -0,0 +1,22 @@
+endpoint: "http://pbs.admatic.com.tr/?host={{.Host}}"
+endpointCompression: gzip
+maintainer:
+  email: "abdulbaki.cam@admatic.com.tr"

```

Comment:
@bakicam  prebid expects maintainer email to be an organization email or mail group but not an email of individual contributor.

`abdulbaki.cam@admatic.com.tr` seems to an email of individual contributor of admatic organization

Reply:
@bakicam  any feedback on above comment

Reply:
Merhaba,
Buradaki committe değişiklikler yapıldı. https://github.com/prebid/prebid-server/pull/3608/commits/f98a6490e6053b5206b233be1d50a3310706880b